Services/GEV/Utils/classes/class.gevOrgUnitUtils.php

```diff
@@ -966,7 +966,36 @@ static public function moveUsers($org_unit_initial_id, $org_unit_final_id) {
 	}
 }
 
+public static function getSuperiorsOfUser($user_id) {
```

Comment:
Bitte auf getDirectSuperiors zurückführen, damit wir den Algorithmus nur einmal pflegen. Bitte Algorithmus für getDirectSuperiors in gevOrgUnitUtils umziehen und in gevUserUtils die Methode aus gevOrgUnitUtils aufrufen.